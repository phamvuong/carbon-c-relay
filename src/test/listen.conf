# listen syntax incarnations
#
# 
# listen
#   <linemode [gzip | bzip2 | xz | ssl]>
#         <<interface[:port] | port> proto <udp | tcp>> ...
#         </ptah/to/file proto unix> ...
#      ;

listen
	linemode
		2013 proto tcp
		0.0.0.0:2013 proto udp
		/var/tmp/tstlisten.sock proto unix
	;

listen
	linemode
		0.0.0.0:2014 proto tcp
	;

listen
	linemode lzma
		0.0.0.0:2015 proto tcp
	;

listen
	linemode ssl
		127.0.0.1:2913 proto tcp
	;
