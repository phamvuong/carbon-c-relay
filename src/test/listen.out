statistics
    submit every 60 seconds
    prefix with carbon.relays.test_hostname
    ;

listen
    linemode
        2013 proto tcp
        0.0.0.0:2013 proto udp
        /var/tmp/tstlisten.sock proto unix
        0.0.0.0:2014 proto tcp
    linemode lzma
        0.0.0.0:2015 proto tcp
    linemode ssl
        127.0.0.1:2913 proto tcp
    ;



